llvm/tools/llvm-cas-object-format/llvm-cas-object-format.cpp

```diff
@@ -639,32 +641,54 @@ void StatCollector::printToOuts(ArrayRef<CASID> TopLevels) {
          << "  => Note: HashSize = " << NumHashBytes << "B\n"
          << "  => Note: PtrSize  = " << sizeof(void *) << "B\n"
          << "  => Note: Cost     = Count*HashSize + PtrSize*Children + Data\n";
-  StringLiteral HeaderFormat = "{0,-22} {1,+10} {2,+7} {3,+10} {4,+7} {5,+10} "
-                               "{6,+7} {7,+10} {8,+7} {9,+10} {10,+7}\n";
-  StringLiteral Format = "{0,-22} {1,+10} {2,+7:P} {3,+10} {4,+7:P} {5,+10} "
-                         "{6,+7:P} {7,+10} {8,+7:P} {9,+10} {10,+7:P}\n";
-  outs() << llvm::formatv(HeaderFormat.begin(), "Kind", "Count", "", "Parents",
-                          "", "Children", "", "Data (B)", "", "Cost (B)", "");
-  outs() << llvm::formatv(HeaderFormat.begin(), "====", "=====", "",
-                          "=======", "", "========", "", "========", "",
-                          "========", "");
-  auto printInfo = [Format, NumHashBytes, &Totals = this->Totals](
-                       StringRef Kind, ObjectKindInfo Info) {
-    if (!Info.Count)
-      return;
-    auto getPercent = [](double N, double D) { return D ? N / D : 0.0; };
-    size_t Size = Info.getTotalSize(NumHashBytes);
-    outs() << llvm::formatv(
-        Format.begin(), Kind, Info.Count, getPercent(Info.Count, Totals.Count),
-        Info.NumParents, getPercent(Info.NumParents, Totals.NumParents),
-        Info.NumChildren, getPercent(Info.NumChildren, Totals.NumChildren),
-        Info.DataSize, getPercent(Info.DataSize, Totals.DataSize), Size,
-        getPercent(Size, Totals.getTotalSize(NumHashBytes)));
-  };
-  for (StringRef Kind : Kinds)
-    printInfo(Kind, Stats.lookup(Kind));
-  printInfo("TOTAL", Totals);
-
+  if(ComputeStatsCSV) {
```

Comment:
LLVM style has a space between `if` and `(`. Please run git-clang-format on the patch to catch other  things as well! (Although, see below, I don't think you need this `if`.)